src/components/mini-pager.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Button } from 'antd';
+import PropTypes from 'prop-types';
+import { BackwardOutlined, CaretLeftOutlined, CaretRightOutlined, ForwardOutlined } from '@ant-design/icons';
+
+function MiniPager({ currentPage, totalPages, onNavigate }) {
+  return (
+    <div className="MiniPager">
+      <Button
+        type="link"
+        className="MiniPager-item"
+        disabled={currentPage <= 1}
+        onClick={() => onNavigate(1)}
+        >
+        <BackwardOutlined />
+      </Button>
+      <Button
+        type="link"
+        className="MiniPager-item"
+        disabled={currentPage <= 1}
+        onClick={() => onNavigate(currentPage - 1)}
+        >
+        <CaretLeftOutlined />
+      </Button>
+      <span
+        className="MiniPager-item"
+        >
+        {currentPage}&nbsp;/&nbsp;{totalPages}
```

Comment:
Small thing, I would find it nicer to leverage the already existing css class and display `flex` with a `gap` 😬

Reply:
Update: Ok, I had missed the slash in there, sorry.